client/signup/steps/site-type/form.jsx

```diff
@@ -40,54 +36,34 @@ class SiteTypeForm extends Component {
 		};
 	}
 
-	handleRadioChange = event => this.setState( { siteType: event.currentTarget.value } );
-
-	handleSubmit = event => {
-		const { siteType } = this.state;
-
-		event.preventDefault();
-
+	handleSubmit = type => {
 		this.props.recordTracksEvent( 'calypso_signup_actions_submit_site_type', {
-			value: siteType,
+			value: type,
 		} );
 
-		this.props.submitForm( siteType );
+		this.setState( { siteType: type } );
+
+		this.props.submitForm( type );
 	};
 
 	renderRadioOptions() {
```

Comment:
Function name does not seem to be relevant anymore.

Reply:
Good catch!